lang/en/tool_objectfs.php

```diff
@@ -30,6 +30,7 @@
 $string['delete_local_objects_task'] = 'Object file system delete local objects task';
 $string['pull_objects_from_storage_task'] = 'Object file system download objects task';
 $string['recover_error_objects_task'] = 'Object error recovery task';
+$string['check_objects_location_task'] = 'Object file system check objects location task';
```

Comment:
I'd just explain this a little more and make it clear that it's only really important for the initial migration.

After that it can be run less frequently like once a night if performance is an issue.I'd also put this into the README in a new section called 'performance tuning'